api/consumer/record-locator/consumer.yaml

```diff
@@ -1358,6 +1360,12 @@ components:
         invalid:
           summary: Unknown
           value: XYZ
+    receiverCode:
```

Comment:
What is receiver code? This is the first time ive heard this phrase 😂

Reply:
Good question - it's how its referred to in the user story hence I've called it this, open to change it, something like `odsCodeExtension`?